nav.css

```diff
@@ -0,0 +1,102 @@
+* {
+	box-sizing: border-box;
+}
+body {
+	margin: 0;
+	padding: 0;
+	font-family: 'Oxygen', sans-serif;
+}
+a {
+	text-decoration: none;
+}
+
+/* navbar*/
+.navbar {
+	background-color: black;
+	box-shadow: 1px 1px 0 black;
+    height: 60px;
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+.navbar-list {
+	display: flex;
+	justify-content: flex-end;
+	padding-left: 25px;
+	width: 50%;
+}
+.navbar-branding {			
+	width: 300px;
+	padding-left: 30px;
+}
+.navbar-branding a {
+	font-size: 20px;
+	color: white;
+}
+.logo2::after {
+	content: "market";
+	color: #669900;
+}
+.logo3 {
+	font-size: 14px;
+	color: white;
+	background-color: #669900;
+	padding: 8px 20px;
+    border-radius: 8px;
+    margin-right: 10px;
+}
+.logo3:hover {
+	background-color: #669968;
+}
+.logo4 {
+	color: #777777;
+	text-transform: capitalize;
+	font-size: 13px;
+}
+.logo4:hover {
+	color: white;
+}
+/* logo-botton*/
+.logo-botton {
+	background-color: white;
+	color: black;
+	height: 80px;
+	display: flex;
+	flex-direction: row;
```

Comment:
Co się stanie jeżeli skasujemy tutaj tą właściwość? Nic. Dlaczego? Niektóre właściwości mają swoje bazowe wartości, np.
`<div>` ma `display: block`
`<span>` ma `display: inline`
`<a>` ma `text-decoration: underline`
podobnie `flex-direction` bazowo ma wartość `row` - zatem nie ma potrzeby deklarować jej jeszcze raz. Swoją drogą to właśnie dlatego jeżeli dajemy elementowi właściwość `display: flex` automatycznie wszystkiego jego bezpośrednie dzieci układają się w rzędzie.